dll/win32/shell32/dialogs/filetypes.cpp

```diff
@@ -1462,129 +1636,166 @@ EditTypeDlgProc(HWND hwndDlg, UINT uMsg, WPARAM wParam, LPARAM lParam)
 static INT CALLBACK
 FileTypesDlg_CompareItems(LPARAM lParam1, LPARAM lParam2, LPARAM lParamSort)
 {
-    PFILE_TYPE_ENTRY Entry1, Entry2;
-    int x;
-
-    Entry1 = (PFILE_TYPE_ENTRY)lParam1;
-    Entry2 = (PFILE_TYPE_ENTRY)lParam2;
-
-    x = wcsicmp(Entry1->FileExtension, Entry2->FileExtension);
-    if (x != 0)
-        return x;
+    PFILE_TYPE_GLOBALS pG = (PFILE_TYPE_GLOBALS)lParamSort;
+    PFILE_TYPE_ENTRY p1 = (PFILE_TYPE_ENTRY)lParam1, p2 = (PFILE_TYPE_ENTRY)lParam2;
```

Comment:
"p1" -> "entry1" and similar for "p2" (original code was more readable in this respect)